website/content/plugins/index.mdx

```diff
@@ -0,0 +1,7 @@
+---
+layout: docs
+page_title: Plugins
+description: FIXME
+---
+
+# FIXME
```

Comment:
@angrycub This is the page's MDX file that I mentioned in slack.
I added basic frontmatter here, and some dummy content.

Feel free to branch-off this PR or commit directly!